packages/dapp-high-roller/src/components/app-provider/app-provider.tsx

```diff
@@ -35,7 +32,7 @@ export class AppProvider {
     this.nodeProvider = params.get("standalone")
       ? new MockNodeProvider()
       : new NodeProvider();
-    return this.nodeProvider.connect().then(this.setupCfProvider.bind(this));
+    return this.nodeProvider.connect().then(this.setupCfProvider.bind(this)); // TODO have this block rendering
```

Comment:
@Alonski I suggest you have a look to how this was implemented on the `dapp-tic-tac-toe` package. Basically you don't render anything until `nodeProvider.isConnected === true`.